### ISSUE TITLE
Renamed files are not being watched

### ISSUE THREAD

**Issue by posti85** [sass/node-sass#1575]
Suppose I have the following files:

_variables.scss

```
$fsize: 12pt;
```

style.scss

```
@import "variables";

div {
    font-size: $fsize;
}
```

Then, I run the command:
`node-sass -w style.scss style.css`

After that, I rename _variables.scss to _variables1.scss, but node-sass doesn't notice about that.

Well, if style.scss is updated (inserting a space), the import error is shown. To make it work again, I update the import reference with the new file name, but if I change the 12pt value to 14pt in variables1.scss, the change isn't neither noticed in the watcher.

**Comment by xzyfer** [comment]
Related https://github.com/sass/node-sass/issues/1538#issuecomment-220158329

**Comment by xzyfer** [comment]
This should be fixed by #1539 which was released in v3.12.0

**Comment by NoelAbrahams** [comment]
@xzyfer, was this released? I'm seem to be having the same problem.

My version:
```
node-sass       4.5.2   (Wrapper)       [JavaScript]
libsass         3.5.0.beta.2    (Sass Compiler) [C/C++]

OS: Windows 10.
```

Basically, when a file is renamed it is no longer watched.
